ssl-impl/src/test/java/io/crate/protocols/ssl/SslConfigurationTest.java

```diff
@@ -179,15 +179,23 @@ public void testExportServerCertChain() throws Exception {
 
         X509Certificate[] x509Certificates = keyStoreSettings.exportServerCertChain();
 
-        assertEquals(2, x509Certificates.length);
-        assertThat(x509Certificates[0].getIssuerDN().getName(), containsString("CN=crate.io"));
-        assertThat(x509Certificates[1].getIssuerDN().getName(), containsString("CN=crate.io"));
-        assertThat(x509Certificates[0].getSubjectDN().getName(), containsString("CN=CrateDB"));
-        assertThat(x509Certificates[0].getSubjectDN().getName(), containsString("PSQL Netty SSL"));
-        assertThat(x509Certificates[0].getSubjectDN().getName(), containsString("L=Dornbirn"));
-        assertThat(x509Certificates[1].getSubjectDN().getName(), containsString("CN=crate.io"));
+        assertThat(x509Certificates.length, is(4));
+        assertThat(x509Certificates[0].getIssuerDN().getName(), containsString("CN=*.crate.io"));
+        assertThat(x509Certificates[1].getIssuerDN().getName(), containsString("CN=*.crate.io"));
+        assertThat(x509Certificates[2].getIssuerDN().getName(), containsString("CN=*.crate.io"));
+        assertThat(x509Certificates[3].getIssuerDN().getName(), containsString("CN=*.crate.io"));
+        assertThat(x509Certificates[0].getSubjectDN().getName(), containsString("CN=localhost"));
```

Comment:
Can't we test this in one line and not with 3 different containsString() assertions?